go-controller/pkg/ovn/policy_test.go

```diff
@@ -729,18 +729,12 @@ var _ = ginkgo.Describe("OVN NetworkPolicy Operations", func() {
 				afterCleanup := initialDB.NBData
```

Comment:
I guess afterCleanup is not used anymore, and can be replaced inline with initialDB.NBData?

Reply:
initialData != afterCleanup as it is modified afterwards

Reply:
right, I am talking about `initialDB.NBData` not `initialData`

Reply:
oh ok, did not realize we were simply talking about the alias. Ok, I can remove that.